l10n_es_aeat_sii/models/account_invoice.py

```diff
@@ -871,7 +947,8 @@ def _process_invoice_for_sii_send(self):
         invoices = self.filtered(
             lambda i: (
                 i.company_id.sii_test or
-                i.period_id.date_start >= SII_START_DATE
+                i.period_id.date_start >= SII_START_DATE or
+                i.sii_registration_key.code in ['16', '14']
```

Comment:
El código 14 para los tipo 'sale' no es de facturas del promer semestre. Lo correcto sería comprobar que sea el codigo '16' y la factura de venta o que sea el codigo '14' y la factura de compra.

Reply:
Deberia ser 'or (i.sii_registration_key.type == 'sale' and i.sii_registration_key.code == '16') or (i.sii_registration_key.type == 'purchase' and i.sii_registration_key.code == '14')